<?xml version="1.0"?>

<launch>
  <arg name="watchdog_enabled" default="false" />
  <arg name="robot_name" default="default" />

	<node pkg="taurob_claw_node" type="taurob_claw_node" name="taurob_claw_node" output="screen">	
		<param name="watchdog" value="$(arg watchdog_enabled)" />
 		<param name="control_enabled_at_startup" value="true" />
		<param name="ip_address" value="10.0.0.50" />
		<param name="port" value="1235" />
		<param name="rot_offset" value="0" />
		<param name="grip_offset" value="0" />
		<remap from="/jointstate_status" to="/joint_states" />
	</node>
	
</launch>
